static/templates/invite_subscription.hbs

```diff
@@ -12,9 +12,9 @@
             <span></span>
             {{#if invite_only}}<i class="fa fa-lock" aria-hidden="true"></i>{{/if}}
             {{#if (eq name ../notifications_stream)}}
-            #{{name}} <i>({{t 'Receives new stream announcements' }})</i>
+            <i class="fa fa-globe" aria-hidden="true"></i> {{name}} <i>({{t 'Receives new stream announcements' }})</i>
             {{else}}
-            #{{name}}
+            {{#if (not invite_only)}}#{{/if}}{{name}}
             {{/if}}
```

Comment:
Can we use a partial to include `static/templates/stream_privacy.hbs` for the icon bit?

Reply:
@timabbott I have used partials to include icons directly from `static/templates/stream_privacy.hbs` in my latest commit.

Reply:
Why can't we use `{{>stream_privacy}}` unconditionally? Is it the space between the hash and the name that's different?

Reply:
@timabbott The problem with directly using `{{>stream_privacy}}` is that it has `<span class="hashtag"></span>` for rendering the hashtag. `span` has been already styled in `components.css` and has a property `border: 1px solid hsla(0, 0%, 0%, 0.6);`.

Here is the screenshot displaying the above problem - 

![bug](https://user-images.githubusercontent.com/96468766/201916160-6f8beeb2-0de4-4b89-ab17-e71f3db780da.png)

One fix I can think of is removing `<span class="hashtag"></span>` and directly using `#` in `stream_privacy.hbs`. 
Then we can directly include `{{>stream_privacy}}` in `invite_subscription.hbs` to render the icons (unconditionally).